pyproject.toml

```diff
@@ -72,7 +72,7 @@ repository = "https://github.com/pypa/setuptools-scm/"
 [project.entry-points."distutils.setup_keywords"]
 use_scm_version = "setuptools_scm._integration.setuptools:version_keyword"
 [project.entry-points."pipx.run"]
-setuptools_scm = "setuptools_scm._cli:main"
+setuptools-scm = "setuptools_scm._cli:main"
```

Comment:
```suggestion
setuptools-scm = "setuptools_scm._cli:main"
setuptools_scm = "setuptools_scm._cli:main"
```

Reply:
Ah ok thanks. I rebased and amended the commit with this change.